src/page/blog/articles-page.jsx

```diff
@@ -19,14 +22,16 @@ export const Articles = () => {
 
   return (
     <div className="max-w-[1440px] mx-auto">
-      <div className="flex mt-10 justify-end mr-32">
-        <a href="/draft" className="underline">
-          Drafts
-        </a>
-        <a href="/articles/create" className="underline ml-5">
-          + New Entry
-        </a>
-      </div>
+      {"id" in currentUser && (
```

Comment:
The `"id" in ` is superfluous, this can be shortened to simply `currentUser &&`.

Reply:
Because currentUser is always an object, it will always return an object and therefore return true in the conditional. 

As per your advice, I have opted to use the isLoggedIn state, and that achieves our end in a cleaner fashion.